Libraries/config/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2015-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+add_library(config SHARED
+            Sources/Config.cpp
```

Comment:
Configurations seem like something any tool could benefit from, so I just made it a separate library.  And it seemed like distinct enough functionality from the rest of xcsdk.